### ISSUE TITLE
All Secretless configuration code samples are updated to reflect new yml config format

### ISSUE THREAD

**Issue by jonahx** [cyberark/secretless-broker#713]
Depends on #708

This story focuses on updating secretless.yml samples in code, demos, tutorials, and documentation. Updating the documentation on the listener / handler concept will happen as part of a separate effort in cyberark/secretless-docs#135.

## Repos to be updated
- [x] Secretless (all samples, docs, demos, etc - tests updated in #712)
- [x] Secretless Docs (all samples & references to config)
- [x] Kubernetes-Conjur-Demo (config used)
- [x] [Sidecar Injector](https://github.com/cyberark/sidecar-injector/blob/master/README.md#secretless-sidecar-injection-example) - README only

# AC:
- [x] All relevant references to secretless.yml are updated to use the new format
- [x] All demos and tutorials are updated to use the new secretless.yml format in documentation and code samples

**Comment by izgeri** [comment]
Note - based on the [design doc](https://github.com/cyberark/secretless-broker/tree/master/design/simpler-configuration.md) we decided to continue supporting the old syntax going forward, so we _do not necessarily need to update all instances of old Secretless config YAML_. In particular, old versions in unused private repos are low priority right now.

**Comment by izgeri** [comment]
There are draft PRs in:
- [Secretless](https://github.com/cyberark/secretless-broker/pull/736)
- [Kubernetes-Conjur-Demo](https://github.com/conjurdemos/kubernetes-conjur-demo/pull/64)
- [Sidecar Injector]()

that can be reviewed now, but should not be merged until the version bump in Secretless is ready.

**Comment by sgnn7** [comment]
@izgeri All PRs look good, just need to wait for merge, real PR opening, and rebuilds.
